spec/rails_helper.rb

```diff
@@ -55,3 +55,31 @@
   config.include FactoryGirl::Syntax::Methods
 
 end
+
+Geocoder.configure(lookup: :test)
```

Comment:
Our build was failing because we hadn't stubbed the calls to the Google API and it was making real requests and it reached a limit, not allowing us to make more requests. So now that it is stubbed, it doesn't make real requests and you don't need to be online to run the test.